### DISCUSSION TITLE
copyright per picture ? per Album ?

### DISCUSSION THREAD

**Discussion by elschman** [LycheeOrg/Lychee#1837] [Q&A] [answered]
Hi,
we (sports club) do have a couple (4,500) of pictures taken from several different photographs. All are in Lychee at our own webspace but we do not own the copyright but we have to state the copyright per picture together with the picture.
Is there any supported way to fulfil this request ?

**Comment by ildyria** [comment, marked as answer]
Unfortunately this is not supported Lychee by at the moment.
It would be a small integration to do, but unfortunately we lack the man power to implement this. :(

**Comment by robs-fotoalbum** [comment]
I would reorganise the photos in albums per event and below sub-albums per photographer, then it should be possible to assign the copyright on an album basis. You just have to ensure not to mix them within an album.
